apps/src/templates/ProjectTemplateWorkspaceIconV2.tsx

```diff
@@ -55,8 +56,10 @@ const ProjectTemplateWorkspaceIconV2: React.FunctionComponent<
       }}
       tooltipOverlayClassName={className}
     >
-      {/* FontAwesomeV6Icon does not work with WithTooltip. */}
-      <i className={'fa-kit fa-connected-level'} />
+      {/* FontAwesomeV6Icon with custom icon does not work with WithTooltip here. */}
```

Comment:
Hey @fisher-alice , I've merged a PR fixing this issue today cause staging branch was closed yesterday (https://github.com/code-dot-org/code-dot-org/pull/61754).
Have you tried pulling staging recently?

Reply:
Thanks Denys - I merged with latest staging and still custom icon still doesn't appear. I swapped out ‘fa-kit fa-connected-level’ with ‘info-circle’  and icon DOES appear. So the issue does seem to be related to the custom icon. 

<img width="491" alt="Screenshot 2024-10-15 at 11 29 23 AM" src="https://github.com/user-attachments/assets/65726a00-9c15-4026-9201-0145c00bb128">

Reply:
Did you add the icon family of `kit`?

Reply:
Hmmm...yes.
This is what I have so far and icon is not being displayed.

```
<button type="button" className={styles.iconButton}>
        <FontAwesomeV6Icon
          iconFamily={'kit'}
          iconName={'fa-connected-level'}
          className={styles.icon}
        />
 </button>
```

Reply:
🎉 It works! I will push the update. Thanks to you both!